l10n_eu_oss_oca/migrations/15.0.2.2.0/post-migration.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2021 Sygel - Manuel Regidor
```

Comment:
I think the copyright is not correct.

Reply:
Yes, this is wrong, I copied it by error. But now I don't have access to this branch because I leaved these organization.

Reply:
In that case you would have to cancel the PR and make the change in one that you have access to so that it is not accepted by mistake.